components/atoms/GoBackLink/GoBackLink.tsx

```diff
@@ -0,0 +1,80 @@
+import { useRouter } from "next/router";
+import styled from "styled-components";
+import { useCharactersContext } from "../../../context/Characters";
+import { createPayload } from "../../../utils";
+
+type ClickEvent = React.MouseEvent;
+interface getQueryCallback {
+  (query: string): number;
+}
+
+const StyledGoBackLink = styled.a`
+  text-decoration: none;
```

Comment:
Como buena práctica Alejandro y yo decimos que las propiedades CSS deben ir ordenadas alfabéticamente, es algo a lo que nos acostumbramos y realmente facilita a la hora de buscar propiedades y evitar duplicidades en ese sentido.

Reply:
Tiene bastante sentido
En otros proyectos que he trabajado se optaba por un order de _parsing_ óptimo...pero claro, eso lo hacían nuestros _robots_.
¿Usáis alguna configuración de _prettier_ para eso?